addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/report/AuthReportData.java

```diff
@@ -32,13 +33,44 @@
 import lombok.Setter;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.parosproxy.paros.Constant;
 import org.zaproxy.addon.authhelper.internal.db.Diagnostic;
 import org.zaproxy.addon.authhelper.internal.db.TableJdo;
 
 @Getter
 @Setter
 public class AuthReportData implements Closeable {
 
+    @Getter
+    public enum FailureDetail {
+        OVERALL(
+                Constant.messages.getString(
+                        "authhelper.authreport.summary.fail.detail.overall.failed")),
+        PASS_COUNT(
+                Constant.messages.getString(
+                        "authhelper.authreport.summary.fail.detail.pass.count.failed")),
+        SESSION_MGMT(
+                Constant.messages.getString(
+                        "authhelper.authreport.summary.fail.detail.sessmgmt.failed")),
+        STATS_FAILURE(
```

Comment:
stats_failure and stats_success might be too generic? We may well decide to fail one other stats in the future.
Also the fact we are using stats shouldnt matter, so maybe `login_failures` ans `no_login_successes`?